pkg/cryptoutils/certificate_test.go

```diff
@@ -126,6 +126,12 @@ func TestCertificatesFromPEM(t *testing.T) {
 
 			expected: []*x509.Certificate{cert1},
 		},
+		{
+			name:     "one cert with trailing newline",
```

Comment:
Can you add a test for two certs in a row with a bonus newline in between?

Reply:
yep, adding test. This worked previously because pem.Decode handles a newline at the start of a PEM block